crates/thetawave_storage/src/core.rs

```diff
@@ -56,9 +57,27 @@ pub(super) fn setup_db(conn: Connection) -> rusqlite::Result<()> {
         PRIMARY KEY (userId, enemyMobType)
     )"
     );
+
+    let create_options_table_sql = format!(
+        "CREATE TABLE IF NOT EXISTS {OPTIONS_TABLE_NAME} (
+        optionsProfileId INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL,
+        bloomEnabled BOOLEAN NOT NULL DEFAULT TRUE,
+        bloomIntensity REAL NOT NULL DEFAULT 1.0
+    )"
+    );
+
     conn.execute(&create_user_stats_sql, []).map(|_| ())?;
     conn.execute(&create_enemies_killed_table_sql, [])
         .map(|_| ())?;
+    conn.execute(&create_options_table_sql, []).map(|_| ())?;
+
+    // insert a default options row if it is not in the db
+    let default_options_row_exists: Option<u32> = conn
+        .query_row("SELECT 1 FROM Options LIMIT 1", [], |row| row.get(0))
+        .optional()?;
+    if default_options_row_exists.is_none() {
+        conn.execute("INSERT INTO Options DEFAULT VALUES", [])?;
```

Comment:
To just make sure a row is always in the table:
```rust
// Ensure that the default options profile is _always_ usable. 
let upsert_default_gameopts_sql = format!("INSERT OR REPLACE INTO {} (optionsProfileId, bloomEnabled, bloomIntensity) VALUES (0, TRUE, 1.0) ON CONFLICT REPLACE", OPTIONS_TABLE_NAME);
conn.execute(upsert_default_gameopts_sql)
```

Reply:
It looks like having "INSERT OR REPLACE" and "ON CONFLICT REPLACE" doesn't work. Looks like having just "INSERT OR REPLACE" is the way to go, let me know if I'm missing something.